css/about-me.css

```diff
@@ -65,6 +65,10 @@
   border-radius: 4px;
 }
 
+.get-my-resume-btn:hover {
+  background: #008552;
+}
+
 @media screen and (min-width: 768px) {
   .about-me-btn {
     background-image: url("../Assets/illustration-about-me-2-desktop.svg");
```

Comment:
- [ ] We are getting very close 🚶 . Invest in little energy further to have a professional portfolio representable during job applications. For this case, I am taking steps to request for few changes to have an amazing portfolio 🥰 . All the changes the previous reviewer suggested have not been fully implemented as intended. However, there are a few changes you applied and have improved the portfolio professionalism slightly 👌 . 
- [ ] Summarizing, If you have noticed, the about-me cards are going beyond the established width of the body. Therefore is suggested you consider fixing the width of `about me` cards. Kindly ensure that the cards are fixed first and then verify again that there is no horizontal scroll anymore. The traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging. Please consider the video to guide you to where the cause of the horizontal scroll might be ☺️ .

The video 😎 


https://user-images.githubusercontent.com/99394374/221381590-7f6fa943-692c-4661-b889-529db8d796ce.mp4